src/shared/containers/SetOrg.tsx

```diff
@@ -255,7 +255,12 @@ const SetOrg: FC = () => {
             component={VariablesIndex}
           />
           {/* Users */}
-          {CLOUD && <Route path={`${orgPath}/users`} component={UsersPage} />}
+          {CLOUD && (
+            <Route
+              path={[`${orgPath}/users`, `${orgPath}/members`]}
+              component={UsersPage}
```

Comment:
can you add a comment here linking to the issue explaining why we have multiple paths?